### DISCUSSION TITLE
Deleting Files from Deta Micros

### DISCUSSION THREAD

**Discussion by MacaylaMarvelous81** [deta/cloud-docs#693] [General] [answered]
Hi! I'm not sure if old files are deleted when I deploy to Micros. This is my current Micro folder structure locally:
![tree1](https://user-images.githubusercontent.com/29156241/209895194-dd81195d-6746-4209-8eab-c860dbac6a3e.png)
(I crossed out files and directories that are ignored by Deta, by default or in .detaignore)
I also deleted some files and directories earlier, but I can see them when I deploy and clone my Micro, my tree looks like this:
![tree2](https://user-images.githubusercontent.com/29156241/209895405-727c2383-f3c6-468c-bc6e-483be5a2b364.png)
Some files that I deleted or ignored earlier are in the new clone... are they still in my Micro's filesystem? If so, how can I remove them?

**Comment by lemonyte** [comment, marked as answer]
This seems to be a common issue, I've had it as well. The most reliable way of deleting things from your Micro is to simply delete the entire Micro and recreate it.

**Reply by MacaylaMarvelous81** [reply]
I see, hope this gets resolved soon :)
